.github/workflows/ci.yml

```diff
@@ -75,6 +75,25 @@ jobs:
 
       - run: cargo test --workspace --features testing-helpers
 
+  no_std:
```

Comment:
@JelteF I've added this CI check, so we can track possible `no_std` regressions on CI.

Reply:
I'm supprised it works, because this issue looks like it would impact this crate: https://github.com/hobofan/cargo-nono/issues/47

Reply:
@JelteF the reason is [explained](https://github.com/hobofan/cargo-nono/issues/47#issuecomment-1197767742):
> there is just no logic in place yet to track and evaluate compiler switches on individual use items (as evaluating arbitrary compiler switches was a rabbit hole I didn't want to go into).

We just have no compiler switches on `use` statements at the moment. So, we're lucky.